packages/iTwinUI-react/src/core/utils/components/MiddleTextTruncation.test.tsx

```diff
@@ -59,3 +59,22 @@ it('should leave original text', () => {
   const containerSpan = container.querySelector('span') as HTMLSpanElement;
   expect(containerSpan.textContent).toBe('No trunc');
 });
+
+it('should render custom text', () => {
+  jest.spyOn(UseOverflow, 'useOverflow').mockReturnValue([jest.fn(), 20]);
+  const text = 'This is some very long text to truncate and expect ellipsis';
+  const { container } = render(
+    <MiddleTextTruncation
+      text={text}
+      textRenderer={(truncatedText) => (
+        <span data-testid='custom-text'>{truncatedText}</span>
```

Comment:
just for fun as your test says `render custom text` try to add some additional chars together with `truncatedText` to check up that content will be rendered.

Reply:
Added some custom text to the end